src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
@adamralph well, the test name itself explains what is being tested, the fixture name here is rather meaningless. It's either this, or adding metadata to the test itself (like Roslyn with `[WorkItem]`).

I'm not particularly bothered by this, though.